### ISSUE TITLE
Bituminous Coal Ore drops Coke

### ISSUE THREAD

**Issue by esotericdesign** [boredherobrine13/morefuelsmod-1.12#1]
Which is intended to be the product of smelting Bituminous Coal, making the latter unobtainable.

**Comment by boredhero** [comment]
Hey! Thanks so much for reporting this issue! I'll hopefully get a patch out for you in a day or two! Sorry I didn't respond sooner. Been a very busy week for me.

**Comment by esotericdesign** [comment]
Thank you as well! TBH I didn't expect any response, thought the mod was abandoned for 1.12.2.

**Comment by boredhero** [comment]
Yeah hey sorry. I'm working on patching it rn. I'm moving the codebase from this repo to the 1.12.x branch of the new repo. I made a [new issue](https://github.com/boredhero/morefuelsmod/issues/8) over there.

Also, nothing that I make is "abandoned" in the strict sense. It's just that I'm very busy with college and I don't really have a ton of time to put into this since it nets me very little income. It's more of a passion project for summers/breaks. But like, any major bugs that pop up, especially something this huge, I'll happily go and patch and release a hotfix version. At some point I *might* backport some of the 1.15.x features back to 1.12.2 but don't get your hopes up. I'm more likely to put time into [my alpha project](https://github.com/boredhero/pv) over here. Plus I still need to make a 1.16.x port forward sometime.

**Comment by boredhero** [comment]
@esotericdesign The new jar should be available [here](https://github.com/boredhero/morefuelsmod/releases/tag/1.12.x-1.6.2-HOTFIX-1)! It should also be available on CurseForge shortly pending staff review of the jar but they're usually pretty quick on this project since it's been around a while now.
